index.html

```diff
@@ -6,7 +6,7 @@
   <meta name="viewport" content="width=device-width, initial-scale=1.0" />
   <title>MACILE News</title>
   <meta name="description"
-    content="“We are the heroes” is the first digital newsletter of MACILE, a non-profit organization based on learning about Math, Science, Engineering and Language." />
+    content="MACILE News is a landing page where newsletters and magazines about projects and activities from MACILE, a non-profit organization based on learning about Math, Science, Engineering and Language, are pointed out." />
```

Comment:
Good job taking a stab at this, it isn't easy. Here's my edit inspired by your words: 

_MACILE News shares the stories from MACILE programs through the years. Learn more about our projects, activities, and students from 2007 to present via the MACILE Magazine and MACILE Bulletin. MACILE is a non-profit organization based on learning about Math, Science, Engineering, and Language._

What do you think? Basic changes

- Avoid pointing out it is a landing page. Landing page is a term used in the world of development.
- Add a call to action for readers. Remember that while these are for bots they also appear for humans
- Avoid complex clauses

I am still not pleased with it. I think it could be a bit tighter. Writing is hard! 

If you have any suggestions, please propose them! Thanks for playing editing volleyball.